packages/cacti-plugin-ledger-connector-stellar/src/main/json/openapi.json

```diff
@@ -86,6 +86,65 @@
         }
       },
 
+      "RunSorobanTransactionRequest": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "contractId",
+          "method",
+          "specXdr",
+          "transactionInvocation"
+        ],
+        "properties": {
+          "contractId": {
+            "type": "string",
+            "pattern": "^C[a-zA-Z0-9]{55}$",
+            "nullable": false,
+            "minLength": 56,
+            "maxLength": 56,
+            "description": "The ID of the contract that was deployed."
+          },
+          "specXdr": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "nullable": false,
+            "description": "Array of strings containing the XDR of the contract specification."
+          },
+          "method": {
+            "type": "string",
+            "nullable": false,
+            "description": "The method to be called on the contract."
+          },
+          "methodArgs": {
+            "type": "object",
+            "nullable": true,
+            "description": "The arguments to pass to the method."
+          },
+          "transactionInvocation": {
+            "$ref": "#/components/schemas/TransactionInvocation",
+            "nullable": false,
+            "description": "The transaction invocation that will be used to run the contract."
+          },
+          "readOnly": {
+            "type": "boolean",
+            "nullable": true,
```

Comment:
@fazatti I'd reconsider making this a mandatory (e.g. `required`) value to ensure that users are explicitly decidigin on whether they'd like to run a transaction with a cost attached to it or not!

Reply:
Thanks @petermetz !
Great suggestion! This will ensure developer's accountability when using this endpoint!
I added the `readOnly` flag as required and adjusted all the tests to reflect this change!

Reply:
@fazzatti Thank you, LGTM! :-)